### PR TITLE
Make VReplication relay log size & rows configurable.

### PR DESCRIPTION
**Pull request by aquarapid**
* Make default relay log size signficantly larger,
  30k -> 250k. Improves performance tables with
  larger rows significantly.
* Fix bug where we were fetching MaxSize instead of
  MaxItems number of rows from VReplication source.

Signed-off-by: Jacques Grove <aquarapid@gmail.com>

### CODE REVIEW

**Review by shlomi-noach — COMMENTED** [review]
:+1: `30k` is clearly exceedingly small.

**Review by rohit-nayak-ps — DISMISSED** [review]
lgtm, but tests need to pass ...
Looks like the failing tests are due to the flakiness around that time when artifact repos were not available and a rerun should fix it

File: go/vt/vttablet/tabletmanager/vreplication/vdbclient.go [hunk 1]
```diff
@@ -92,8 +92,8 @@ func (vc *vdbClient) ExecuteFetch(query string, maxrows int) (*sqltypes.Result,
 
 // Execute is ExecuteFetch without the maxrows.
 func (vc *vdbClient) Execute(query string) (*sqltypes.Result, error) {
-	// Number of rows should never exceed relayLogMaxSize.
-	return vc.ExecuteFetch(query, relayLogMaxSize)
+	// Number of rows should never exceed relayLogMaxItems.
+	return vc.ExecuteFetch(query, *relayLogMaxItems)
```
Comment (deepthi) [inline go/vt/vttablet/tabletmanager/vreplication/vdbclient.go:96]: Before this PR, the allowed number of rows was 30K. With the changes in this PR, that defaults to 1K (though the new flag let us increase the limit). @rohit-nayak-ps is this an acceptable change?

Comment (rohit-nayak-ps) [reply go/vt/vttablet/tabletmanager/vreplication/vdbclient.go:96]: Good catch. @aquarapid  should we increase this? Currently for tables with row lengths < 250bytes the 1K max row limit will kick in before the max buffer size is reached.

Comment (aquarapid) [reply go/vt/vttablet/tabletmanager/vreplication/vdbclient.go:96]: OK, pushing it to 5k.

Comment (deepthi) [reply go/vt/vttablet/tabletmanager/vreplication/vdbclient.go:96]: @rohit-nayak-ps is that a good number?

Comment (rohit-nayak-ps) [reply go/vt/vttablet/tabletmanager/vreplication/vdbclient.go:96]: yes, should be a good default

**Review by rohit-nayak-ps — APPROVED** [review]
lgtm

**Pull request merged**